web/core/components/issues/issue-detail/issue-activity/activity-filter.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, Fragment } from "react";
+import { observer } from "mobx-react";
+import { Check, ListFilter } from "lucide-react";
+import { Popover, Transition } from "@headlessui/react";
+// ui
+import { Button } from "@plane/ui";
+// constants
+import { ACTIVITY_FILTER_TYPE_OPTIONS, EActivityFilterType } from "@/constants/issue";
+// helper
+import { cn } from "@/helpers/common.helper";
+
+type Props = {
+  selectedFilters: EActivityFilterType[];
+  toggleFilter: (filter: EActivityFilterType) => void;
+};
+
+export const ActivityFilter: FC<Props> = observer((props) => {
+  const { selectedFilters, toggleFilter } = props;
+  return (
+    <Popover as="div" className="relative">
```

Comment:
We have a component for this now. Please use it.

Reply:
That component is not flexible for use here, so I need to create a global filter popover component. I have created a ticket WEB-1894 for this task and will take care of it.